Makefile.in

```diff
@@ -404,13 +404,11 @@ clean-unittest:
 	@echo "    CLEAN ocean_forcing/tests"
 	@cd ocean_forcing/tests;   $(MAKE) clean
 
-clean-all-tests: clean-test     
+clean-all-tests: clean-test
 	@echo "    CLEAN parallel/special/long tests"
-	@PYTHONPATH=python tools/testharness.py --parallelism=parallel --clean >/dev/null
 	@PYTHONPATH=python tools/testharness.py --length=special --clean >/dev/null
-	@PYTHONPATH=python tools/testharness.py --length=special --parallelism=parallel --clean >/dev/null
```

Comment:
The 'special' tests might be worth removing at this point. The tag was introduced as a 'do not run, this is broken' option, and they've not been tested in a long time, so could probably go. If anyone really cares about them I think they'd have fixed them some time in the past decade :-)